src/main/java/io/lbry/lbrynet/MainActivity.java

```diff
@@ -74,6 +92,30 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
             }
         }
     }
+    
+    @Override
+    public void onRequestPermissionsResult(int requestCode, String permissions[], int[] grantResults) {
+        switch (requestCode) {
```

Comment:
Should this method do something if the `requestCode` doesn't match any declared case? If that's the case, a default case should be added to the switch statement.

Reply:
We only care about the request code for the storage permission, so there is no need to handle any other request code.